comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -341,6 +341,10 @@ func (ia *inventoryagent) fetchSystemProbeMetadata() {
 
 	ia.data["feature_gpu_monitoring_enabled"] = sysProbeConf.GetBool("gpu_monitoring.enabled")
 
+	// Synthetics Test module / system-probe
+
+	ia.data["feature_synthetics_collector_enabled"] = sysProbeConf.GetBool("synthetics.collector.enabled")
```

Comment:
I think this is not correct. `synthetics.collector.enabled` is a config from datadog.yaml, not system-probe.yaml.

So, you might want to move it to `fetchCoreAgentMetadata()` and use the right config item.